src/main.rs

```diff
@@ -56,7 +56,7 @@ fn main() -> amethyst::Result<()> {
 
     let pipe = Pipeline::build().with_stage(
         Stage::with_backbuffer()
-            .clear_target([0.002, 0.01, 0.01, 1.0], 1.0)
+            .clear_target([0.02, 0.15, 0.02, 1.0], 1.0)
```

Comment:
Were you editing this line for debugging reasons? 😅

Reply:
You mean the background color ? I just got tired of looking at a black bacground. Should I put it back ?